### DISCUSSION TITLE
table_type FORIEGN Postgres

### DISCUSSION THREAD

**Discussion by akshayjain3450** [readysettech/readyset#1204] [Q&A] [answered]
Hi @aglover @alanamarzoev ,

I want to know if readyset supports foreign table_type in PostgresDB. Also, does it support Tables in custom schema like 'xyz' instead of 'public' for caching?

Thank You!

**Comment by aglover** [comment, marked as answer]
Hi @akshayjain3450! Readyset does not support foreign tables. Readyset does support custom schemas.
